.github/workflows/export-dynamic.yaml

```diff
@@ -34,13 +25,17 @@ on:
         required: false
         type: string
 
-  push:
-    tags:
-      - "v*.*.*"
+      plugins-repo:
+        description: Target Community Plugins repository to export plugins from
+        type: string
+        required: false
+        default: ''
 
-  pull_request:
-    branches:
-      - 'releases/**'
+      overlay-branch:
+        description: Branch of the overlay structure (current branch by default).
+        type: string
+        required: false
+        default: ''
 
 concurrency:
```

Comment:
This `concurrency` block should probably be removed, since it's a `workflow-call` workflow